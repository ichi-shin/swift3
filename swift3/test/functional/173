#!/bin/bash
# Copyright(c)2014 NTT corp. All Rights Reserved.
# DB1-1-4-2_1

. ./common
S3USER=tester2
_s3_put /bucket
_s3_post /bucket/obj?uploads -D - | _filter_curl xml
echo '<VersioningConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/">
<Status>Enabled</Status>
</VersioningConfiguration>' > $tmp.versioning.xml
_s3_put /bucket?versioning -T $tmp.versioning.xml

_sw_get /

S3USER=tester
_s3_get / -D - | _filter_curl xml
