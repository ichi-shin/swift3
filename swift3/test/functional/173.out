QA output created by 173
> s3curl --id tester2 -- -X PUT http://SWIFT_HOST/bucket... 200
> s3curl --id tester2 -- -X POST -D - http://SWIFT_HOST/bucket/obj?uploads... 200
HTTP/1.1 200 OK
Content-Length: LENGTH
Content-Type: application/xml
Date: DATE
x-amz-id-2: TXID
x-amz-request-id: TXID
X-Trans-Id: TXID

<?xml version='1.0' encoding='UTF-8'?>
<InitiateMultipartUploadResult xmlns="http://s3.amazonaws.com/doc/2006-03-01/">
  <Bucket>bucket</Bucket>
  <Key>obj</Key>
  <UploadId>UPLOAD_ID</UploadId>
</InitiateMultipartUploadResult>
> s3curl --id tester2 -- -X PUT -T /TMP.versioning.xml http://SWIFT_HOST/bucket?versioning... 200
> curl -X GET http://SWIFT_HOST/v1/TENANT/... 200
bucket
bucket+segments
bucket+versions
> s3curl --id tester -- -X GET -D - http://SWIFT_HOST/... 200
HTTP/1.1 200 OK
Content-Length: LENGTH
Content-Type: application/xml
Date: DATE
x-amz-id-2: TXID
x-amz-request-id: TXID
X-Trans-Id: TXID

<?xml version='1.0' encoding='UTF-8'?>
<ListAllMyBucketsResult xmlns="http://s3.amazonaws.com/doc/2006-03-01/">
  <Owner>
    <ID>TESTER</ID>
    <DisplayName>TESTER</DisplayName>
  </Owner>
  <Buckets/>
</ListAllMyBucketsResult>
